House-of-glass/src/router/Router.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Routes,Route } from 'react-router-dom';
+import { MockApi } from '../pages/api-response';
+import Cart from '../pages/cart/Cart';
+import Landing from '../pages/landing/Landing';
+import Products from '../pages/products/Listing/Products';
+import NotFound from '../pages/utils/NotFound/NotFound';
+import Wishlist from '../pages/wishlist/Wishlist';
```

Comment:
You could make it cleaner by importing all the components in index.js and from there importing all the components using single import statement.

Reply:
Thank You Vaibhav, Have corrected them in the latest commit.